app/components/Settings/Settings.react.js

```diff
@@ -6,14 +6,12 @@ import ReactToolTip from 'react-tooltip';
 import classnames from 'classnames';
 import * as Actions from '../../actions/sessions';
 import fetch from 'isomorphic-fetch';
-import SplitPane from 'react-split-pane';
 import {Tab, Tabs, TabList, TabPanel} from 'react-tabs';
 import ConnectionTabs from './Tabs/Tabs.react';
 import UserConnections from './UserConnections/UserConnections.react';
 import DialectSelector from './DialectSelector/DialectSelector.react';
 import ConnectButton from './ConnectButton/ConnectButton.react';
 import Preview from './Preview/Preview.react';
-import TableTree from './Preview/TableTree.react.js';
```

Comment:
Is this file no longer in use?  I was working on some jest tests for table tree.  If it is not needed I can move on to some other tests

Reply:
yes, it's still in use (tests are always welcome :smile:). I moved the import to `Preview.react.js`. Now `Preview.react.js` contains all the components displayed under the `QUERY` tab.